bin.src/validateDrp.py

```diff
@@ -25,32 +25,34 @@
 import os.path
 import sys
 
-from lsst.validate.drp import checkAstrometryPhotometry
+from lsst.validate.drp import validate
 
 
-def defaultData(repo):
-    # List of visits to be considered
-    visits = [849375, 850587]
-
-    # List of CCD to be considered (source calalogs will be concateneted)
-    ccd = [12, 13, 14, 21, 22, 23]
-    filter = 'r'
-
-    # Reference values for the median astrometric scatter and the number of matches
-    good_mag_limit = 21.0
-    medianRef = 25
-    matchRef = 5000
+if __name__ == "__main__":
```

Comment:
Is it worth using `argparse` here? What you've got is hardly excessively complicated, but when you're loading multiple things from `sys.argv`, printing help messages, etc, it might be a useful simplification.